<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddFriend</class>
 <widget class="QWidget" name="AddFriend">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>356</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>456</width>
    <height>356</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>456</width>
    <height>356</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>456</width>
    <height>356</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="addFriendWidget" native="true">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>3</y>
     <width>450</width>
     <height>350</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>450</width>
     <height>350</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>450</width>
     <height>350</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>450</width>
     <height>350</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#addFriendWidget{
	background-color: rgb(255, 255, 255);
}</string>
   </property>
   <widget class="QWidget" name="topWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>450</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>450</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>450</width>
      <height>30</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>450</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#topWidget{
	background-color: rgb(240, 241, 245);
}</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>5</y>
       <width>120</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>120</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	font-size: 14px;
	background-color: rgb(240, 241, 245);
}</string>
     </property>
     <property name="text">
      <string>xxx 添加好友</string>
     </property>
    </widget>
    <widget class="QPushButton" name="closeBtn">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>5</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	image: url(:/Icon/close2.png);
	border-radius: 0;
	background-color: rgba(255, 255, 255, 0);
}
QPushButton:hover{
	image: url(:/Icon/close.png);
	background-color: rgb(255, 0, 0);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="miniBtn">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>5</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	image: url(:/Icon/mini2.png);
	border-radius: 0;
	background-color: rgba(255, 255, 255, 0);
}
QPushButton:hover{
	image: url(:/Icon/mini.png);
	background-color: rgb(136, 136, 136);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="infoWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>130</width>
      <height>285</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>130</width>
      <height>285</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>130</width>
      <height>285</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>130</width>
      <height>285</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#infoWidget{
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0.0454545 rgba(255, 255, 255, 255), stop:0.545455 rgba(193, 232, 250, 255), stop:0.835227 rgba(198, 232, 250, 255), stop:0.926136 rgba(255, 255, 255, 255));
}</string>
    </property>
    <widget class="QLabel" name="sexLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	color: rgb(133, 136, 138);
}</string>
     </property>
     <property name="text">
      <string>性别：-</string>
     </property>
    </widget>
    <widget class="QLabel" name="headLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>110</width>
       <height>110</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>110</width>
       <height>110</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>110</width>
       <height>110</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>110</width>
       <height>110</height>
      </size>
     </property>
     <property name="text">
      <string>头像</string>
     </property>
    </widget>
    <widget class="QPushButton" name="usernameBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border: none;
}</string>
     </property>
     <property name="text">
      <string>12345678910</string>
     </property>
    </widget>
    <widget class="QLabel" name="ageLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	color: rgb(133, 136, 138);
}</string>
     </property>
     <property name="text">
      <string>年龄：-</string>
     </property>
    </widget>
    <widget class="QLabel" name="placeLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	color: rgb(133, 136, 138);
}</string>
     </property>
     <property name="text">
      <string>所在地：-</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>315</y>
      <width>450</width>
      <height>35</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>450</width>
      <height>35</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>450</width>
      <height>35</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>450</width>
      <height>35</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
	background-color: rgb(229, 239, 247);
}</string>
    </property>
    <widget class="QPushButton" name="close2Btn">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>5</y>
       <width>75</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font-size: 12px;
	color: rgb(85, 85, 85);
	background-color: rgb(255, 255, 255);
	border-radius: 2px;
	border: 1px solid rgb(204, 204, 204);
}
QPushButton:hover{
	background-color: rgb(242, 242, 242);
}</string>
     </property>
     <property name="text">
      <string>关闭</string>
     </property>
    </widget>
    <widget class="QPushButton" name="nextBtn">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>5</y>
       <width>75</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font-size: 12px;
	color: rgb(85, 85, 85);
	background-color: rgb(255, 255, 255);
	border-radius: 2px;
	border: 1px solid rgb(1, 143, 251);
}
QPushButton:hover{
	background-color: rgb(242, 242, 242);
}</string>
     </property>
     <property name="text">
      <string>下一步</string>
     </property>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>320</width>
      <height>285</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>320</width>
      <height>285</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>320</width>
      <height>285</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>320</width>
      <height>285</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#checkInfoPage{
	background-color: rgb(255, 255, 255);
}
QWidget#setInfoPage{
	background-color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="checkInfoPage">
     <property name="minimumSize">
      <size>
       <width>320</width>
       <height>285</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>320</width>
       <height>285</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>320</width>
       <height>285</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QTextEdit" name="infoTextEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>280</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTextEdit{
    border-radius: 5px;
    font-size: 13px;
    color: black;
	background-color: rgb(255, 255, 255);
    border: 1px solid #D1D1D1;
}
QTextEdit:hover{
	border: 1px solid #1583DD;
}
</string>
      </property>
      <property name="placeholderText">
       <string>我是...</string>
      </property>
     </widget>
     <widget class="QLabel" name="infoLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>请输入验证信息：</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="setInfoPage">
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTextBrowser{
	border: none;
}</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun';&quot;&gt;您的好友添加请求已经发送成功，正在等待对方确认。&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>close2Btn</sender>
   <signal>clicked()</signal>
   <receiver>AddFriend</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>397</x>
     <y>331</y>
    </hint>
    <hint type="destinationlabel">
     <x>600</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeBtn</sender>
   <signal>clicked()</signal>
   <receiver>AddFriend</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>433</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>642</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
